XenModel/Utils/Helpers.Versions.cs

```diff
@@ -450,6 +450,12 @@ public static bool XapiEqualOrGreater_22_33_0(IXenConnection conn)
             return coordinator == null || ProductVersionCompare(coordinator.GetXapiVersion(), "22.33.0") >= 0;
         }
 
+        public static bool XapiEqualOrGreater_23_10_0(IXenConnection conn)
+        {
+            var coordinator = GetCoordinator(conn);
+            return coordinator == null || ProductVersionCompare(coordinator.GetXapiVersion(), "23.10.0") >= 0;
+        }
+

```

Comment:
Not relevant to this PR but maybe for the future: looks like we're just replicating code here. We should probably have `XapiEqualOrGreater(IXenConnection connection, XapiVersion version)` with a list of version enums encoding all versions used (or something similar). This would still allow us to easily refactor existing calls. Can be done another time, just wanted to add my 2 cents on the matter.

Reply:
I agree, things will get out of hand soon.